catalog/transformer/catalog-transformer-zip/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -42,6 +42,8 @@
             <entry key="shortname" value="zipC"/>
             <entry key="mime-type" value="application/zip"/>
             <entry key="schema" value="urn:catalog:metacard"/>
+            <entry key="export-resultset" value="false"/>
+            <entry key="displayName" value="ZIP"/>
```

Comment:
This is the same issue with `Preview HTML`. All of the other display names sound like options but `Export` and `Preview` sound like actions.

Reply:
`Backup` is the other idea but it also sounds like an action.  `Signed ZIP`?

Reply:
`Signed ZIP` sounds good to me. Ultimately this may need to be removed unless I establish a default export type with the PR I linked. Another potential ticket that has spawned from expanding our export capabilities is an admin configurable whitelist/blacklist of metacard and query response transformers for these lists.

We do something similar already for query response transformers with a service property. 

https://github.com/codice/ddf/pull/4240